config/katello.yaml

```diff
@@ -9,6 +9,9 @@
   :installer_dir: "."
   :default_values_dir: "/tmp"
   :module_dirs: ["./_build/modules", "../katello-installer/modules"]
+  :parser_cache_path:
+    - /usr/share/foreman-installer/parser_cache/foreman.yaml
+    - ./_build/parser_cache/katello.yaml
```

Comment:
If you note, this is the only other place that uses `_build` [1] and I believe it was there to aide with local testing. But you will notice that the second argument in the array is more in line with my suggestion. And keeps things universally working.

[1] https://github.com/Katello/katello-installer/pull/382/files#diff-f7223b52c32ec607907f68076d4a0db3R11